src/tests/api/test_sendmail.py

```diff
@@ -129,6 +129,9 @@ def test_sendmail_rule_create_minimal(token_client, organizer, event):
             'subject': {'en': 'meow'},
             'template': {'en': 'creative text here'},
             'send_date': '2018-03-17T13:31Z',
+            "restrict_to_status": ['p', 'valid_if_pending'],
+            # maybe this (the default) should be set when not giving any options?
+            # but then it's not entirely clear elsewhere…
```

Comment:
What do you mean exactly? Yes, if the field is missing, the default must be used.

Is this then a TODO that should be resolved merging?